plugin-jdbc-arrow-flight/src/main/java/io/kestra/plugin/jdbc/arrowflight/Query.java

```diff
@@ -29,21 +29,25 @@
 @Plugin(
     examples = {
         @Example(
-            title = "Send a SQL query to a database and fetch a row as outputs using Apache Arrow Flight SQL driver",
+            title = "Send a SQL query to a database and fetch row(s) using Apache Arrow Flight SQL driver",
             code = {
+                "id: arrow-flight-sql-query",
+                "type: io.kestra.plugin.jdbc.arrowflight.Query",
```

Comment:
This is automatically added via our documentation generation tool for non-full examples.
Please remove them

Reply:
Ohh, I see. I will remove this.